utils/podman/Makefile

```diff
@@ -82,14 +80,48 @@ else
   CNIFILE:=87-podman-bridge.conflist
 endif
 
+MAKE_VARS += \
+	GO_INSTALL_BIN_PATH="$(strip $(GO_PKG_INSTALL_BIN_PATH))" \
+	BUILD_DIR="$(PKG_BUILD_DIR)" \
+	GO_BUILD_DIR="$(GO_PKG_BUILD_DIR)" \
+	GO_BUILD_BIN_DIR="$(GO_PKG_BUILD_BIN_DIR)" \
+	GO_BUILD_DEPENDS_PATH="$(GO_PKG_BUILD_DEPENDS_PATH)" \
+	GO_BUILD_DEPENDS_SRC="$(GO_PKG_BUILD_DEPENDS_SRC)" \
+	GOOS="$(GO_OS)" \
+	GOARCH="$(GO_ARCH)" \
+	CC="$(TARGET_CC)" \
+	CXX="$(TARGET_CXX)" \
+	CGO_CFLAGS="$(filter-out $(GO_CFLAGS_TO_REMOVE),$(TARGET_CFLAGS))" \
+	CGO_CPPFLAGS="$(TARGET_CPPFLAGS)" \
+	CGO_CXXFLAGS="$(filter-out $(GO_CFLAGS_TO_REMOVE),$(TARGET_CXXFLAGS))" \
+	CGO_LDFLAGS="$(TARGET_LDFLAGS)" \
+	GOPATH="$(GO_PKG_BUILD_DIR)" \
+	GOCACHE="$(GO_BUILD_CACHE_DIR)" \
+	GOMODCACHE="$(GO_MOD_CACHE_DIR)" \
+	GOFLAGS="$(GO_PKG_GCFLAGS)" \
+	GO_PKG_CFLAGS="$(GO_PKG_CFLAGS)" \
+	CGO_ENABLED=1 \
+	GOENV=off \
+	PREFIX=/usr \
+	LIBEXECDIR=/usr/lib \
+	LIBEXECPODMAN=/usr/lib/podman \
+	SHAREDIR_CONTAINERS=/usr/share/containers \
+	ETCDIR=/etc \
+	TMPFILESDIR=/var/run/podman \
+	BUILDTAGS="$(GO_PKG_TAGS)" \
+	EXTRA_LDFLAGS="$(GO_PKG_LDFLAGS)"
+
 define Build/Prepare
 	$(call Build/Prepare/Default)
 	$(eval $(call Download,default-registries))
 	$(eval $(call Download,default-policy))
 endef
 
 define Package/podman/install
-	$(call GoPackage/Package/Install/Bin,$(1))
+	#$(call GoPackage/Package/Install/Bin,$(1))
```

Comment:
Please remove this line rather than commenting it out.

Reply:
@dangowrt

Thanks, my bad :) I removed it. Please, review it again :)